backend/app/routes.py

```diff
@@ -40,7 +60,8 @@ def node(node_id):
 #shell function - outputs json for use on frontend
 @app.route('/link-nodes/<from_node_id>/<to_node_id>', methods=['GET'])
 def get_links_between_two_nodes(from_node_id, to_node_id):
-    """Returns links of the shortest path between two nodes on the HERE network"""
+    """Returns links of the shortest path between two nodes on the HERE network,

```

Comment:
It might be nice to just indicate which function in the code is actually finding the shortest path.